File: 2942_words_containing_char/words_containing_char.js
// JS: https://leetcode.com/problems/find-words-containing-character/submissions/1178251694/
// TS: https://leetcode.com/problems/find-words-containing-character/submissions/1178253035/

function findWordsContainingChar(words, char) {
    let indexes = [];

    for(let i = 0; i < words.length; i++) {
        if(words[i].includes(char)) {
            indexes.push(i)
        }
    }

    return indexes;
}

console.log(findWordsContainingChar(["leet","code"], 'e'));

File: 2942_words_containing_char/words_containing_char.py
# Py3: https://leetcode.com/problems/find-words-containing-character/submissions/1178253850/

def find_words_contianing_char(words, char):
    indexes = []

    for i, word in enumerate(words):
        if char in word:
            indexes.append(i)
    
    return indexes

print(find_words_contianing_char(["leet","code"], 'e'));